### PR TITLE
CUMULUS-1913: Add datepicker to Reconciliation Reports

### PR DESCRIPTION
**Pull request by jennyhliu**
**Summary:** Summary of changes

Addresses [CUMULUS-1913: Add datepicker to Reconciliation Reports](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1913)

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: app/src/js/components/locale.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ export const strings = new LocalizedStrings({
     logo: 'cumulus-logo.png',
     operations: 'Operations',
     pdrs: 'Pdrs',
+    reconciliation_reports: 'Reconciliation Reports',
```
Comment (flamingbear) [inline app/src/js/components/locale.js:37]: If we add it here do we need to have it in the gitc block as well?  I think so, but I'm not 100% certain.

Comment (flamingbear) [inline app/src/js/components/locale.js:37]: If we add it here do we need to have it in the gitc block as well? I think so, but I'm not 100% certain.

Comment (jennyhliu) [reply app/src/js/components/locale.js:37]: Thanks, updated gitc block

Comment (flamingbear) [reply app/src/js/components/locale.js:37]: lgtm

**Comment by jjmccoy** [comment]
UI/UX: The width of the heading is too large with the DTP so the double line is fine. Everything else looks and works fine. LGTG!

**Pull request merged**